src/bgworker/mod.rs

```diff
@@ -44,6 +53,9 @@ pub fn main() {
         }
         log::error!("Panickied. Info: {:?}. Backtrace: {}.", info, backtrace);
     }));
+    std::alloc::set_alloc_error_hook(|layout| {
```

Comment:
I have seen this API has been removed at https://github.com/rust-lang/rust/pull/109507 , and `OOM` is a regular panic from that.

Could https://doc.rust-lang.org/std/panic/fn.set_hook.html worked?

Reply:
If the pull request is merged, we will just remove set_alloc_hook and replace panic layload type to the std one. It's simple if we bump the compiler version someday.

Reply:
> If the pull request is merged, we will just remove set_alloc_hook and replace panic layload type to the std one. It's simple if we bump the compiler version someday.

Fine, Let's comment a TODO/Note at here to remind it.